updater/lib/dependabot/analyse_files_command.rb

```diff
@@ -0,0 +1,152 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "base64"
+require "json"
+require "dependabot/base_command"
+require "dependabot/dependency_snapshot"
+require "dependabot/errors"
+require "dependabot/opentelemetry"
+require "dependabot/updater"
+require "github_api/dependency_submission"
+
+module Dependabot
+  class AnalyseFilesCommand < BaseCommand
```

Comment:
Naming is hard, right out the gate I've realised this name invites 🇺🇸 vs 🇬🇧 spelling confusion.

I am open to renaming this, this is just how we've talked about this so far.

Reply:
I'd be more concerned though about what to call the Dependabot CLI command since that is the user facing side that we shouldn't change after choosing it.

Reply:
suggestions:

- detect_files_command
- parse_files_command (wait does this exist already?)
- update_graph_command

I'm kinda leaning towards parse or update_graph tbh